dipper/sources/OMIM.py

```diff
@@ -471,58 +474,70 @@ def _transform_entry(self, e, graph, globaltt):
             elif omimtype == self.globaltt['gene']:
                 if abbrev is not None:
                     nodelabel = abbrev
+                #  G_omim subclass_of  gene
                 model.addClassToGraph(
-                    omim_curie, nodelabel, omimtype, newlabel)  # Do subclass G
-            elif omimtype == self.globaltt['Phenotype']:
-                model.addClassToGraph(
-                    omim_curie, newlabel, description='Phenotype')  # Don't subclass P
+                    omim_curie, nodelabel, omimtype, newlabel)
+            elif omimtype == self.globaltt['obsolete']:
+                # X_omim subclass_of RIP
+                model.addClassToGraph(omim_curie, newlabel, omimtype)
             else:
-                model.addClassToGraph(omim_curie, newlabel, omimtype)  # Do subclass X?
+                # omim NOT subclass_of D|P|or ?...
+                model.addClassToGraph(omim_curie, newlabel)
 
             # add the original screaming-caps OMIM label as a synonym
-            model.addSynonym(omim_curie, label)
+            # P.R. says these are not an exact synonym in #714
+            model.addSynonym(omim_curie, label, model.globaltt['has_related_synonym'])
```

Comment:
I don't think we need to add synonym data here, but it's not a big deal either way